ci/constraints-test/pydantic-v1-install.txt

```diff
@@ -0,0 +1,3 @@
+# this is higher than our min version but is the earliest version that actually works
+# TODO: update our min version or fix gx for our current min version
+pydantic==1.10.8
```

Comment:
Our min version of pydantic in our requirements file is `1.9` but gx actually breaks at import time on this version, and it looks like we were never testing against this version in our CI.
This does not appear related to moving to pydantic v2.